dowhy/interpreters/confounder_distribution_intepreter.py

```diff
@@ -0,0 +1,58 @@
+import seaborn as sns
+import numpy as np
+import matplotlib.pyplot as plt
+
+from dowhy.interpreters.visual_interpreter import VisualInterpreter
+from dowhy.causal_estimators.propensity_score_weighting_estimator import PropensityScoreWeightingEstimator
+from dowhy.causal_estimator import CausalEstimate
+
+
+class ConfounderDistributionIntepreter(VisualInterpreter):
+    SUPPORTED_ESTIMATORS = [PropensityScoreWeightingEstimator, ]
+
+    def __init__(self, estimate, **kwargs):
+        super().__init__(estimate, **kwargs)
+        if not isinstance(estimate, CausalEstimate):
+            error_msg = "The interpreter method expects a CausalEstimate object."
+            self.logger.error(error_msg)
+            raise ValueError(error_msg)
+        self.estimator = self.estimate.estimator
+        if not any(isinstance(self.estimator, est_class) for est_class in ConfounderDistributionIntepreter.SUPPORTED_ESTIMATORS):
+            error_msg = "The interpreter method only supports propensity score weighting estimator."
+            self.logger.error(error_msg)
+            raise ValueError(error_msg)
+
+    def interpret(self, fig_size=(10, 7), font_size=14):
```

Comment:
With the fix above, no need to define parameters for `interpret()` method, can directly use `self.fig_size` and other variables initialized in the `__init__` method above.